spec/integration/rspec/eq_matcher_spec.rb

```diff
@@ -953,4 +953,8 @@
   it_behaves_like "a matcher that supports elided diffs" do
     let(:matcher) { :eq }
   end
+
+  it_behaves_like "a toggleable key" do
```

Comment:
The context here is the matcher, so saying that the matcher is a toggleable key sounds a bit awkward. What if this were "a matcher that supports a toggleable key"?